cswinrt/strings/additions/Windows.Storage.Streams/StreamOperationsImplementation.cs

```diff
@@ -201,8 +201,9 @@ internal static class StreamOperationsImplementation
                     int buffSize = 0x4000;
                     if (bytesToWrite < buffSize)
                         buffSize = (int)bytesToWrite;
-
-                    await dataStream.CopyToAsync(stream, buffSize, cancelToken).ConfigureAwait(continueOnCapturedContext: false);
+                    
+                    if (buffSize > 0)
```

Comment:
is the buffSize check needed?  what happens without it?

Reply:
Without it, the crash in the bug this PR is fixing will hit  I believe [this](https://docs.microsoft.com/en-us/dotnet/api/system.io.stream.copytoasync?view=net-5.0#System_IO_Stream_CopyToAsync_System_IO_Stream_System_Int32_System_Threading_CancellationToken_) is datastream.CopyToAsync which indicates it throws when zero.